Server/Services/LeaderBoardService.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Server.Controllers;
```

Comment:
It would be great to remove that dependency. I don't have an IDE open, but if it's using only `LeaderBoardPlayer` it can be just moved to that file.